pkg/common/DnnImageFeaturizer.targets

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this should be a `.props` file instead.

The difference is that a `.props` file gets imported BEFORE an importing .csproj gets executed. Which means that if a user wanted to modify the `Content` item below, they can in their .csproj without writing a `<Target>`

Reply:
Fair enough, changed it, thanks!

---
In reply to: [236760593](https://github.com/dotnet/machinelearning/pull/1447#discussion_r236760593) [](ancestors = 236760593)